include/dolphin/OS/OSThread.h

```diff
@@ -0,0 +1,129 @@
+#ifndef OS_THREAD_H
+#define OS_THREAD_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "dolphin/os/OSContext.h"
```

Comment:
Typo here, should be OS instead of os

Reply:
That would make it incompatible with other projects that use Nintendo's naming scheme, Nintendo used `dolphin/os` rather than `dolphin/OS`